lib/definitions.php

```diff
@@ -1022,6 +1022,7 @@
     0   => trans('existing'),
     1   => trans('under construction'),
     2   => trans('planned'),
+    3   => trans('suspended'),
 );
```

Comment:
A jak to przenieść do raportu UKE? CHyba UKE nie onsługuje proponowanego stanu w ramach raportów.

Reply:
W raporcie UKE może się dla tego typu pokazywać stan 1.

Reply:
inny pomysł: "suspended" -> "disabled" i nie meldujmy w raporcie UKE?